Adyen/Model/PrefilledShopperInformation.swift

```diff
@@ -41,14 +65,14 @@ public struct PrefilledShopperInformation: ShopperInformation {
     ///   - card: Shopper's card basic information, optional.
     public init(shopperName: ShopperName? = nil,
```

Comment:
Maybe we need another `init` with `phoneNumber`. And old `init` we can deprecate

Reply:
You are right. I have fixed this 9d6daef2e7cb0abbbe6701564d59fa85bc35c9dc